src/Planner/CollectColumnIndetifiers.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
Need to rename this file into `CollectColumnIdentifiers.h` from `CollectColumnIndetifiers.h`.